src/bitmap/store/bitmap_store.rs

```diff
@@ -403,12 +410,12 @@ impl<B: Borrow<[u64; BITMAP_LENGTH]>> DoubleEndedIterator for BitmapIter<B> {
 
 #[inline]
 pub fn key(index: u16) -> usize {
-    index as usize / 64
+    usize::from(index) / 64
 }
 
 #[inline]
-pub fn bit(index: u16) -> usize {
-    index as usize % 64
+pub fn bit(index: u16) -> u32 {
+    u32::from(index) % 64
 }
```

Comment:
Is it a necessary change? Could you remove that from this PR, please?

Reply:
Ah, right, not anymore, it was when using `checked_shl`, which takes a u32. Removed.